call-notes/call_6.md

```diff
@@ -0,0 +1,160 @@
+
+So, I have a few links. One is a [Google doc](https://docs.google.com/document/d/1RkNAyz-5aKvv5FF44b8SoKifChKB705y5SdcEoqMPIc/edit?usp=sharing). I started in this when the proposal was just starting. So it contains many details and a lot of designs we rejected. And why? Because I am currently modifying both the SIMD draft and this Google doc. So this Google doc might be outdated and so some design choices might be different if that's the case SIMD is the newest proposal. And the actual SIMD draft here as well. And this is why I'm asking whether you have any questions again. 
+
+So, I haven't described the details of the algorithm. I have backup slide at the end to talk about. There are basically two algorithms, one is `how we calculate which blocks I should repair when I have all the LastVotedForkSlots`. And the other is `how do I select a block in the HeaviestFork`. So these two algorithms are in the SIMD and I also have some description here.
+
+#### Questions
+
+Video |  [22:59](https://youtu.be/GA5AVg_svj8?t=1379)
+-|-
+
+**Wen Xu:** Let me know if you have questions.
+
+**Mango Team:** Can you hear me?
+
+**Jacob Creech:** Yes, we can hear you.
+
+**Wen Xu:** Yep, yep!
+
+**Mango Team:** Yeah, so actually I am from Mango team and like we are currently implementing [SIMD 47](https://github.com/solana-foundation/solana-improvement-documents/blob/main/proposals/0047-syscall-and-sysvar-for-last-restart-slot.md) which is like the last restart slot. So,  I am wondering if this silent repair stage should be considered also a restart slot?
+
+**Wen Xu:** Sorry I haven't checked that document in detail. So I think, correct me if I am wrong, your proposal is to expose the last restarted slot somehow through Ledger right?
+
+**Mango Team:** Yeah, exactly.
+
+**Wen Xu:** Yeah so, here I think it is orthogonal but once the silent repair phase is over and we know where we are restarting from. We could also like connect to your code and expose this information somewhere right?
+
+**Mango Team:** Oh yeah exactly, so like silent repair should be considered as a restart right? Because it may take time to restart the slot silently right? That's my question.
+
+**Wen Xu:** So when it is in the silent repair phase, we don't know for sure it's a restart. They are negotiating but they don't know whether they can decide on the same block. So in that case I don't think we will expose anything. And once that phase is over and we know we are really entering the restart, then we can expose that restart slot. Did that answer the question?
+
+**Mango Team:** Okay! yeah, it makes sense yeah.
+
+**Anonymous:** There shouldn't actually be any changes to that proposal because once we've committed to restarting at the coordinated restart slot there is going to be a hard Fork anyway and we will. So that will update this as far.
```

Comment:
```suggestion
**Anonymous:** There shouldn't actually be any changes to that proposal because once we've committed to restarting at the coordinated restart slot there is going to be a hard fork anyway and we will. So that will update the Sysvar.
```